core/Cargo.toml

```diff
@@ -59,9 +59,10 @@ serde_json = "1.0"
 prost-build = "0.11"
 
 [features]
-default = [ "secp256k1", "ecdsa" ]
+default = [ "secp256k1", "ecdsa", "rsa" ]
```

Comment:
In the changelog, we say people need to enable the feature for `rsa` support but we enable it as a default feature here. Given that we are already making a breaking change, I think I'd prefer to not enable this as the default.

Reply:
Ok, removed